### PR TITLE
bump packages and unlock consistent eval

### PR DESCRIPTION
**Pull request by vix633**

### CODE REVIEW

**Comment by vix633** [comment]
FYI @BraydonKains @sarahboyd

File: package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "d2l-activity-exemptions": "Brightspace/d2l-activity-exemptions#semver:^2",
     "d2l-awards-leaderboard-ui": "Brightspace/awards-leaderboard-ui#semver:^1",
     "d2l-button-group": "BrightspaceUI/button-group#semver:^3",
-    "d2l-consistent-evaluation": "BrightspaceHypermediaComponents/consistent-evaluation#v0.0.22",
+    "d2l-consistent-evaluation": "BrightspaceHypermediaComponents/consistent-evaluation#semver:^0.0",
```
Comment (jeroach) [inline package.json:63]: Any reason this is `0.0` instead of just `0`?

Comment (vix633) [reply package.json:63]: Just the way that semver likes it when you're below v1

**Comment by vix633** [comment]
Wont be bumping anything here, will only unlock consistent eval

**Pull request merged**